src/dropdown/dropdown.ts

```diff
@@ -1,12 +1,12 @@
 import {Directive, Input, HostListener} from 'angular2/core';
 
-@Directive({selector: '[ngb-dropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
+@Directive({selector: '[ngbDropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
```

Comment:
Can't we leave a selector as `ngb-dropdown`? I'm not saying we should, just didn't test so don't know if technically possible...